src/DataDefinitionsBundle/EventListener/PimcoreSettingsListener.php

```diff
@@ -24,7 +24,6 @@ class PimcoreSettingsListener
 {
     public function indexSettings(IndexActionSettingsEvent $settingsEvent): void
     {
-        $settingsEvent->addSetting('data-definitions-import-definition-writeable', (new ImportDefinition())->isWriteable());
-        $settingsEvent->addSetting('data-definitions-export-definition-writeable', (new ExportDefinition())->isWriteable());
+
```

Comment:
Ah, just found the issue with this: it is used for the Add button, it gets disabled without this.

Reply:
issue?

Reply:
The "Add" button (new import/export profile) will be disabled when not in debug because it relies on this to be present.

Reply:
is that a problem?

Reply:
Yes because you can't add new profiles even though they're writable? You can edit existing ones.